addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -228,11 +236,26 @@ options.registry.blog_cover = options.Class.extend({
 
         _.each(this.$el, function (el) {
             var $el = $(el);
-            $el.toggleClass('d-none',
-                $el.is(':not([data-change])') && !self.$target.hasClass('cover')
-                || $el.is(':has([data-select-class])') && self.$target.hasClass('o_list_cover'));
+
+            if ($el.is(':not([data-change])')) {
+                $el.removeClass('d-none');
+
+                _.each(['size','filters','text_size', 'text_align'], function (opt_name) {
+                    var $opts = $el.find('[data-coverOpt="'+ opt_name +'"]');
```

Comment:
- `data-cover-opt`, no camelcase in the xml
- We can now use ES6 as we do not support IE anymore and we can thus avoid using underscore and `self` variable when needed, here:
```javascript
['size', 'filters', 'text_size', 'text_align'].forEach(optName => {
   this.doStuff();
});
```
I am not a fan of all ES6 features, but this is great